codegen/builtin_fs.go

```diff
@@ -1019,6 +1118,11 @@ func (dot DownloadDockerTarball) Call(ctx context.Context, cln *client.Client, r
 		return err
 	}
 
+	dockerAPI, _ := DockerAPI(ctx)
+	if dockerAPI.Moby {
+		return errdefs.WithDockerEngineUnsupported(ProgramCounter(ctx))
```

Comment:
Seems surprising that moby would not support exporting the tarball, without buildkit I think this is equivalent to `docker_ save`?

Reply:
Not much we can do unless we contribute to engine, I think its just a matter of no one needing it in buildx yet.